NebulaWorld/Factory/BeltManager.cs

```diff
@@ -1,11 +1,16 @@
-﻿using NebulaModel.Packets.Belt;
+﻿using HarmonyLib;
+using NebulaModel.Packets.Belt;
 using System.Collections.Generic;
 
 namespace NebulaWorld.Factory
 {
     public static class BeltManager
     {
         public static List<BeltUpdate> BeltUpdates = new List<BeltUpdate>();
+
+        public static readonly AccessTools.FieldRef<object, BeltRenderingBatch[]> GetCargoTraffic =
```

Comment:
Wouldn't this be better to be surfaced as an extension method on top of `CargoTraffic`?

Reply:
Also, this should be named GetRenderingBatches